src/reactnative/builds/algoliasearch.reactnative.js

```diff
@@ -24,6 +24,12 @@ function algoliasearch(applicationID, apiKey, opts) {
 
   opts._ua = opts._ua || algoliasearch.ua;
 
+  opts.timeouts = opts.timeouts || {
+    connect: 2 * 1000,
+    read: 4 * 1000,
```

Comment:
Maybe this should have the same values as the swift client? https://github.com/algolia/algoliasearch-client-swift/blob/master/Source/AbstractClient.swift#L124